recipes/grpc/all/conanfile.py

```diff
@@ -82,27 +76,21 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            try:
-                del self.options.fPIC
-            except Exception:
-                pass
+            self.options.rm_safe("fPIC")
```

Comment:
Then, we should use this: `required_conan_version = ">=1.53.0"`

Reply:
Good catch!! thanks :D